### DISCUSSION TITLE
How to download DCF models?

### DISCUSSION THREAD

**Discussion by anbe6083** [OpenBB-finance/OpenBB#968] [Q&A] [answered]
Hi there, I ran the DCF command and received this message:
`Analysis ran for AAPL
Please look in /home/python/exports/excel/AAPL 2021-11-21.xlsx for the file.`

How/where do I access these files? I'm not familiar with Docker. Thanks!

**Comment by aia** [comment, marked as answer]
You want to follow these instructions:

https://stackoverflow.com/questions/22049212/docker-copying-files-from-docker-container-to-host
